UI-extra/price.dust

```diff
@@ -0,0 +1,13 @@
+<div class="price">
+    {?oldPrice}
+        <s class="grey-color">{oldprice}</s>
+        <span>{currentprice}</span>
+        {?savePrice}
```

Comment:
If `oldPrice` and `savePrice` is true. We'll see something like... 
![screen shot 2016-08-03 at 4 21 14 pm](https://cloud.githubusercontent.com/assets/4543395/17385244/53fb171c-5996-11e6-9856-340a3f2635cc.png)